### DISCUSSION TITLE
Thanks for all your hard work!

### DISCUSSION THREAD

**Discussion by CamSoper** [BigThunderSR/homeassistant-addons-onstar2mqtt#817] [General]
Hey @BigThunderSR, this project would have died if you hadn't forked it and nursed it through several key-related issues. I want to express my gratitude, so I sponsored you. It's not much, but I hope you enjoy a nice Starbucks or something.

Another thanks to @metheos too! Ya'll are awesome!

**Comment by BigThunderSR** [comment]
Thanks much @CamSoper!
